src/program/lwaftr/setup.lua

```diff
@@ -58,6 +63,9 @@ function lwaftr_app(c, conf)
       error("One or both of the 'next_hop' values must be specified")
    end
 
+   -- Validate configuration file we've been given.
+   validate_config(conf)
+
```

Comment:
Hm, you're right I think. I'll try and think of a better place to put it, I need a place which works for the initial validation for unreconfigurable and reconfigurable instances of the lwaftr alike.

Good catch.